stripe/_api_requestor.py

```diff
@@ -78,6 +79,16 @@ def __init__(
             self._client = stripe.default_http_client
             self._default_proxy = proxy
 
+        if stripe.default_http_client_async:
+            self._client_async = stripe.default_http_client_async
+        else:
+            stripe.default_http_client_async = (
+                _http_client.new_default_http_client_async(
+                    verify_ssl_certs=verify, proxy=proxy
+                )
+            )
+            self._client_async = stripe.default_http_client_async
```

Comment:
Wouldn't 
```suggestion
        if not stripe.default_http_client_async:
            stripe.default_http_client_async = (
                _http_client.new_default_http_client_async(
                    verify_ssl_certs=verify, proxy=proxy
                )
            )
        self._client_async = stripe.default_http_client_async
```
be easier to read?

Reply:
I do like that better!